src/styles/buy.scss

```diff
@@ -0,0 +1,19 @@
+.buy {
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 12px 70px;
+  color: rgba(255, 255, 255, 1);
+  background-color: $blue-accent;
```

Comment:
The variable `$blue-accent` is used here and is correctly defined in `variables.scss` as `rgba(0, 172, 220, 1)`.